### ISSUE TITLE
Missing C2 commands

### ISSUE THREAD

**Issue by ligfx** [openc2e/openc2e#173]
According to the [Creatures 2 CAOS Language Guide](https://web.archive.org/web/20071015154335/http://www.gamewaredevelopment.co.uk/downloads/cdn/C2CAOSGuide.pdf), openc2e is missing the following C2 commands:
- [ ] AIM
- [ ] BBT2
- [ ] CAMN
- [ ] CBRG
- [ ] CBRX
- [ ] CLS2
- [ ] CREA
- [ ] DDE: HATC
- [ ] DOOR
- [ ] FIRE
- [ ] GRND
- [ ] HOUR
- [ ] INS#
- [ ] ISAR
- [ ] LACB
- [ ] LANG
- [ ] LCUS
- [ ] LNG
- [ ] MESG SHOU
- [ ] MESG SIGN
- [ ] MESG TACT
- [ ] MINS
- [ ] MUSC
- [ ] NEID
- [ ] NEW: SCEN
- [ ] OBDT
- [ ] OBSV
- [ ] ORGN
- [ ] PAUS
- [ ] POIN
- [ ] PRLD
- [ ] PUHL
- [ ] PUPT
- [ ] RAIN
- [ ] RCLR
- [ ] RMS#
- [ ] RMSC
- [ ] RNDR
- [ ] SAY#
- [ ] SCOR
- [ ] SNDQ
- [ ] STM#
- [ ] TECO
- [ ] WLDH
- [ ] WLDW

**Comment by bedalton** [comment]
I was also going through the C1 and C2 commands yesterday, and found missing commands in both, as well as some commands marked as variant 'all', or some marked C1 and C2, that are not actually available in the runtime.

These are missing commands from C2. Should I post a separate issue for missing C1 commands? Should I list commands that are erroneously included, or does it not matter to the engine?

BBD: VCB1 wordListIndex (integer) vocabListIndex (integer)
DDE: CELL lobe (integer) cell (integer) dentype (integer)
DDE: GENE
DDE: GETB organ (integer)
DDE: GETB ALLR property (integer)
DDE: GETB BIOC
DDE: GETB CNAM
DDE: GETB CTIM
DDE: GETB DATA
DDE: GETB EMTR organ (integer)
DDE: GETB LVOB family (integer) genus (integer) species (integer)
DDE: GETB MONK
DDE: GETB NEWV
DDE: GETB ORGN
DDE: GETB OVVD
DDE: GETB RCTN organ (integer)
DDE: GETB RPTY room (integer)
DDE: GETB RRCT
DDE: GIDS FMLY family (integer)
DDE: GIDS GNUS family (integer) gnus (integer)
DDE: GIDS ROOT
DDE: GIDS SPCS family (integer) genus (integer) species (integer)
DDE: LCUS type (integer) organ (integer) tissue (integer) locusid (integer)
DDE: LNEU flags (integer)
DDE: LOBE
DDE: NACT
DDE: PANC
DDE: PIC2 width (integer) height (integer) filename [string]
DDE: PUTB string [string] option (token)
DDE: SCRP family (integer) genus (integer) species (integer) event (integer)
DDE: WORD index (integer)
EDIT
EDIT
LNG+
NEW: CBTN spriteFile (token) numimages (integer) firstImage (integer) plane (integer)
NEW: LIFT spriteFile (token) numimages (integer) firstImage (integer)
NEW: RAIN spriteFile (token) numimages (integer) firstImage (integer) plane (integer) clone (integer)
RTYP
SYS: ABRT
SYS: EDIT left (integer) top (integer) right (integer) bottom (integer)
SYS: QUIT
SYS: WPOS x (integer) y (integer) width (integer) height (integer)
SYS: WRLD filename [string]


I actually entered all the commands I could into a database to generate stubs for my C# Unity version of OpenC2e. At the time I didn't know people were also working on OpenC2e again. Writing commands to a DB allowed me to generate method stubs for all CAOS commands, and code to read all required variables and assign generic out values. It also allowed me to write preprocessor stubs to exclude non-variant commands, as I was only working on C1.

So should I create a separate issue for missing commands in other variants? Should I also list code such as MODU, ELIF, DCOR, etc that are listed as all and not actually valid in C1 and C2. There is a longer list including all DBG: commands.

**Comment by ligfx** [comment]
@bedalton yes, please create a separate issue per variant for missing commands, and another separate issue per variant for extra commands that shouldn't be valid.

**Comment by bedalton** [comment]
So it turns out the documentation for one command is wrong.
`DDE: LCUS (command) type (integer) organ (integer) tissue (integer) locusid (integer)`

does not actually exist. It is actually:
`LCUS (integer) type (integer) organ (integer) tissue (integer) locusid (integer)`
